api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -469,4 +466,82 @@ public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() throws Ex
 		new VisitValidator().validate(visit, errors);
 		assertEquals(true, errors.hasFieldErrors("voidReason"));
 	}
+
+	/**
+	 *  @see VisitValidator#validate(Object,Errors)
+	 */
+	@Test
+	@Verifies(value= "should fail validation if the visit start date is before the birth date of the patient", method = "validate(Object, Errors)")
+	public void validate_shouldFailValidationIfVisitStartDateIsBeforeBirthdate() throws Exception {
+		Calendar calendar = Calendar.getInstance();
+		calendar.set(1970, Calendar.FEBRUARY, 2);
+		Visit visit = makeVisit(42);
+		visit.setStartDatetime(calendar.getTime());
```

Comment:
It's good practice actually to include a sanity check that ensures that actually the visit start date is before the birtdate just in case the test data is edited.